test/project_types/script/commands/enable_test.rb

```diff
@@ -220,6 +219,23 @@ def test_calls_application_enable_with_configuration_file_and_properties_overrid
         end
       end
 
+      def test_calls_application_enable_with_invalid_configuration_file
+        ::FakeFS.with_fresh do
```

Comment:
I thought we changed it to `include TestHelper::FakeFS` and not doing this `.with_refresh` explicitly any more?

Reply:
I had used this approach in my change rather than the include to make it so FakeFS is only used where it's needed to limit the scope (and thus the risk for breaking things unrelated to the change).